core/backend/src/internal/workspace/WorkspaceImpl.ts

```diff
@@ -555,10 +559,16 @@ class EditorContainerImpl extends WorkspaceContainerImpl implements EditableWork
   public override getWorkspaceDb(props: WorkspaceDbProps): EditableWorkspaceDb {
     return this.getEditableDb(props);
   }
+
   public getEditableDb(props: WorkspaceDbProps): EditableWorkspaceDb {
     const db = this._wsDbs.get(workspaceDbNameWithDefault(props.dbName)) as EditableDbImpl | undefined ?? new EditableDbImpl(props, this);
-    if (this.cloudContainer && this.cloudContainer.queryDatabase(db.dbFileName)?.state !== "copied")
+    const isPrerelease = semver.major(db.version) === 0 || semver.prerelease(db.version);
+
+    if (!isPrerelease && this.cloudContainer && this.cloudContainer.queryDatabase(db.dbFileName)?.state !== "copied") {
+      this._wsDbs.clear();
```

Comment:
not sure about this, but in my tests i was having a problem where `new EditableDbImpl` would add the uneditable db to this._wsDbs, which would cause problems later